lib/utils/index.js

```diff
@@ -605,6 +605,22 @@ class Utils {
       throw err;
     }
   }
+
+  static isSafari(desiredCapabilities = {}) {
+    const {browserName} = desiredCapabilities;
+
+    if (browserName && browserName.toLowerCase() === 'safari') {
+      return true;
+    }
+
+    return false;
+  }
+
+  static isLocalhost(webdriver = {}) {
+    const {host} = webdriver;
+    
+    return ['127.0.0.1', 'localhost'].indexOf(host) > -1;
+  }
```

Comment:
If we are adding this can we move some of the logic from `client.js` like client.api.isSafari to use this util method?

Reply:
done